main.tf

```diff
@@ -167,9 +170,9 @@ data "aws_iam_policy_document" "deployment" {
   statement {
     actions = var.deployment_actions
 
-    resources = flatten(distinct([
-      local.origin_bucket.arn,
-      formatlist("${local.origin_bucket.arn}%s*", each.value),
+    resources = distinct(flatten([
+      [local.origin_bucket.arn],
+      formatlist("${local.origin_bucket.arn}/%s*", each.value),
```

Comment:
This is a better way of ensuring the bucket policy is valid, as we're handling the slash programatically now instead of deferring it to the user.